src/modules/auth/reducer.js

```diff
@@ -2,6 +2,8 @@ import { types } from './actions'
 
 const initialState = {
 	userinfo: null,
+	allowedTenants: null,
+	currentAllowedTenant: null,
```

Comment:
Just `tenants` ... don't know what is a difference among `allowedTenants` and `currentAllowedTenant` ... and there should not be any.

Reply:
@ateska Well, I have renamed it in the last commit to `tenants` and `current` - to give you a small exmplanation, the `tenants` are all tenants, which are authorized for the user to be used in the application (this will not provide you `userinfo`, but it is exctracted from `rbac`). The `current` is the current tenant, which is taken from the url parameters (if defined) or it is the first authorized tenant from the list of authorized tenants. This is handy, when user will log out from the application and in the url will stay the tenant parameter. But when e.g. another user without the authorization for this particular tenant will try to log in, it will not let him in or it will redirect to the part of the application, for which the user has authorized access (by another tenant).

Reply:
Ok, as commented elsewhere, it is then `activeTenant` ...